control/frdata.py

```diff
@@ -753,6 +768,35 @@ def feedback(self, other=1, sign=-1):
 
         return FRD(fresp, other.omega, smooth=(self.ifunc is not None))
 
+    def append(self, other):
+        """Append a second model to the present model.
+
+        The second model is converted to FRD if necessary, inputs and
+        outputs are appended and their order is preserved"""
+        other = _convert_to_frd(other, omega=self.omega, inputs=other.ninputs,
+                                outputs=other.noutputs)
+
+        # TODO: handle omega re-mapping
+
+        new_fresp = np.zeros(
+            (
+                self.noutputs + other.noutputs,
+                self.ninputs + other.ninputs,
+                self.omega.shape[-1],
+            ),
+            dtype=complex,
+        )
```

Comment:
This is OK, but a different style than elsewhere in the codebase.  Normally we just wrap things in a way that uses less than 78 characters/line and uses as few lines as possible.  So something like:
```
        new_fresp = np.zeros(
            (self.noutputs + other.noutputs, self.ninputs + other.ninputs,
             self.omega.shape[-1]), dtype=complex)